components/org.wso2.carbon.identity.scim2.common/src/main/java/org/wso2/carbon/identity/scim2/common/impl/SCIMUserManager.java

```diff
@@ -731,6 +727,32 @@ public User updateUser(User user, Map<String, Boolean> requiredAttributes) throw
         return getDetailedUsers(userNames, requiredAttributes);
     }
 
+    /**
+     * Method to resolve the domain name.
+     *
+     * @param domainName Domain to run the filter
+     * @param node       Expression node for single attribute filtering
+     * @return Resolved domainName
+     * @throws CharonException
+     */
+    private String getDomainName(String domainName, ExpressionNode node) throws CharonException {
```

Comment:
Shall we rename this method to `resolveDomainName` to reflect it's functionality?

Reply:
Fixed